frigate/video.py

```diff
@@ -556,7 +597,7 @@ def process_frames(
                         box = obj[2]
```

Comment:
Just a heads up.
I noticed that for the latest aarch64 build this:
```
                for index in idxs:
                    obj = group[index[0]]
``` does not work anymore because index became an int32. 
Locally I updated the code to the 
```
                for index in idxs:
                    index = index if isinstance(index, np.int32) else index[0]
                    obj = group[index]
```

Reply:
And this is because you rebuilt wheels? I don't typically rebuild wheels between releases unless I have to.

Reply:
Yup, I did rebuild the weels after checking out the code and this line never worked for me.

Reply:
Can you avoid rebuilding? Changes to the underlying libraries often create a bunch of sleeper issues and I don't like to change them alongside new features.

Reply:
@yury-sannikov this is broken in amd64 too. Is there a PR for a fix anywhere or does this going to get reverted?

Reply:
I am planning to revert that change.